setup.cfg

```diff
@@ -12,7 +12,7 @@ tag_prefix =
 parentdir_prefix = modin-
 
 [tool:pytest]
-addopts = --disable-pytest-warnings --cov-config=setup.cfg --cov=modin --cov-append --cov-report=xml
+addopts = --disable-pytest-warnings --cov-config=setup.cfg --cov=modin --cov-append
```

Comment:
Now coverage reports are merged manually, this requires using the raw format (simply `.coverage`).